src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -458,7 +463,7 @@ public NuGetVersion Version
 
         public string GetVersionString()
         {
-            return string.Join(".", _version.Major, _version.Minor, _version.Patch);
```

Comment:
I wanted to include the prerelease label. Will do some more testing to make sure we don't break.